tests/Main.fs

```diff
@@ -422,7 +416,7 @@ Expecting a datetime but instead got: undefined
 
         it "Fetch.get works with unit response" <| fun d ->
             promise {
-                let! res = Fetch.get("http://localhost:3000/get/unit")
+                let! res = Fetch.get<_,_>("http://localhost:3000/get/unit")
```

Comment:
Removing `<_,_>` will brake syntax highlighting in ionide: 
![image](https://user-images.githubusercontent.com/5693190/66375822-93841900-e9ae-11e9-8661-2363d6ddaa99.png)